client/src/app/site/pages/meetings/pages/participants/pages/participant-import/services/participant-import.service/participant-import.service.ts

```diff
@@ -140,8 +140,11 @@ export class ParticipantImportService extends BaseUserImportService {
         const result = await this.presenter.call({
             permissionRelatedId: this.activeMeetingId,
             searchCriteria: entries.map(entry => {
-                const username = !!entry.username ? entry.username : `${entry.first_name} ${entry.last_name}`;
-                return { username, email: entry.email };
+                if (entry.username) {
+                    return { username: entry.username };
+                }
+
+                return { username: `${entry.first_name} ${entry.last_name}`, email: entry.email };
```

Comment:
```suggestion
                return { username: `${entry.first_name}${entry.last_name}`, email: entry.email };
```

Since https://github.com/OpenSlides/openslides-backend/pull/1458, usernames are generated without a space in them, so the search should not include it either.